bin/run_docker.py

```diff
@@ -180,6 +185,21 @@ def run_docker(submission_id: str, log_file_name: str = "docker.log") -> None:
         log_file_name=log_file_name, log_file_path=log_file_path, log_text=log_text
     )
 
+    # Rename the predictions file if requested
+    if rename_output:
+        # Get the output directory based on the mounted volumes dictionary used to run the container
+        output_dir = next((key for key in volumes.keys() if "output" in volumes[key]["bind"]), None)
+        if not os.path.exists(output_dir):
+            raise ValueError("Output directory found in ``volumes`` does not exist.")
+
+        # Get the predictions file, assuming the file exists. Otherwise, ``predictions_file = None``
+        predictions_file = next((file for file in glob(os.path.join(output_dir, "predictions.*")) if os.path.exists(file)), None)
+        if predictions_file is None:
+            raise ValueError("No predictions file found in output directory.")
```

Comment:
Isn't this something that we handle in the validate step? If we throw an error here the submitter won't be notified of the issue, right?

Reply:
Although I guess the Nextflow process also would fail if the predictions file were missing...

Reply:
I'll merge this and continue this conversation in the other PR.